atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/sweep/SweepTaskRunnerImpl.java

```diff
@@ -116,6 +122,17 @@ public SweepResults run(TableReference tableRef, int batchSize, @Nullable byte[]
             return SweepResults.createEmptySweepResult(0L);
         }
 
+        try {
```

Comment:
might want to extract the try/catch to separate method `runWithDeleteLock` as `run` is getting long